cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -18,6 +18,7 @@ package v1alpha1
 
 import (
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/kubeapiserver/authorizer"
```

Comment:
This pulls pulls in internal informers (allowed since its an api server) which pulls in internal clients.  This package is included in client-go because it got registered `testapi`, which causes the client-go update scripts to choke.

I plan to duplicate the constant here.